framework/api_vulkan_sample.cpp

```diff
@@ -304,6 +304,7 @@ void ApiVulkanSample::input_event(const vkb::InputEvent &input_event)
 						{
 							get_gui().visible = !get_gui().visible;
 						}
+						break;
```

Comment:
Would you mind to add this `break` to hpp_api_vulkan_sample.cpp as well?

Reply:
Also applied to the hpp variant.